src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/Apprenticeship.cs

```diff
@@ -661,7 +661,20 @@ private void ResolveDatalocks(DateTime stopDate)
                 return ConfirmationStatus.Unconfirmed;
             }
 
-            return ApprenticeshipConfirmationStatus.ConfirmationStatus;
+            return DisplayConfirmationStatus(Email, 
+                ApprenticeshipConfirmationStatus.ApprenticeshipConfirmedOn, 
+                ApprenticeshipConfirmationStatus.CommitmentsApprovedOn, 
+                ApprenticeshipConfirmationStatus.ConfirmationOverdueOn);
+        }
+
+        public ConfirmationStatus? DisplayConfirmationStatus(string Email, DateTime? confirmedOn, DateTime approvedOn, DateTime? overdueOn)
+        {
```

Comment:
don't think we need approvedOn (and I doubt it will be needed in the future), Also we can get the original function DisplayConfirmationStatus() to call this explicitly